server/data/educationAndWorkPlanClient.test.ts

```diff
@@ -101,7 +101,7 @@ describe('educationAndWorkPlanClient', () => {
         userMessage: 'An unexpected error occurred',
         developerMessage: 'An unexpected error occurred',
       }
-      educationAndWorkPlanApi.get(`/action-plans/${prisonNumber}`).reply(500, expectedResponseBody)
+      educationAndWorkPlanApi.get(`/action-plans/${prisonNumber}`).thrice().reply(500, expectedResponseBody)
```

Comment:
Ah ha, to account for the automatic retries ? 👍

Reply:
Yep, exactly.